UK/Data/Plugin/TopSky_NERC/TopSkySettings.txt

```diff
@@ -218,6 +218,45 @@ WXR_Echo_dBZ_Hi=20
 WXR_Echo_dBZ_Med=15
 WXR_Echo_dBZ_Lo=12 
 
+[LON_]
+Setup_ScreenLat=DTY
```

Comment:
With this being first, will it not match any starting with LON and set DTY?

i.e. if I log in as LON_N, this will match first?

Reply:
I've tested and the order doesn't matter